hypershift-operator/controllers/nodepool/inplace.go

```diff
@@ -201,7 +202,8 @@ func (r *NodePoolReconciler) reconcileMachineSet(ctx context.Context,
 		})
 	}
 
-	if message != "" && isUpdatingConfig {
+	if isUpdatingConfig {
```

Comment:
why are we doing any different to the check above isUpdatingVersion here?

The message is being set here https://github.com/openshift/hypershift/pull/1978/files#diff-186fca173b913fab90bc6a26071e1255532f14ce6156a796e04de919d8e97c13R180

So even discriminate what is set in that annotation by config change / version change
Or just set a neutral message in that annotation and add a prefix here for both isUpdatingVersion  /  isUpdatingConfig